README.md

```diff
@@ -16,9 +16,9 @@ Every 10 minutes the metrics agent creates a tarball of the gathered metrics and
 
 ### Kubernetes Versions
 
-#### 1.27 and below
+#### 1.28 and below
 
-Kubernetes versions 1.27 and below are supported by the metrics agent on AWS cloud service (EKS).
+Kubernetes versions 1.28 and below are supported by the metrics agent on AWS cloud service (EKS).
 
 #### 1.26 and below
```

Comment:
Pretty sure we also support GKE and AKS 1.27. Would need to double check with Bala though to make sure that is correct.

Reply:
Confirmed with Bala that we support GKE and AKS 1.27. Updated README.